pkgs/development/python-modules/unsloth/default.nix

```diff
@@ -78,13 +100,51 @@ buildPythonPackage rec {
     "torch"
   ];
 
+  # Provide C compiler at runtime for Triton JIT.
+  propagatedBuildInputs = [
+    stdenv.cc
```

Comment:
```
$ nix eval -f '<nixpkgs>' --argstr crossSystem aarch64-linux stdenv.cc.name
nix-repl> __splicedPackages.stdenv.cc ? __spliced
false
```

Not the compiler you want to propagate. Besides, maybe better not to force the compiler into the closure (I don't disagree the UX is problemtic)